renderer/MapRenderer.py

```diff
@@ -0,0 +1,38 @@
+import mapnik
+
+
+class MapRenderer:
+    def __init__(self):
+        # style = '/var/map-styles/ridewithgps-tilemill/ridewithgps-tilemill.xml'
+        self.style = '/vagrant/renderer/rwgps.xml'
```

Comment:
This is resolved.
Should have just shipped with prod. Setting up dev environment is a pain in the ass.